packages/server/graphql/mutations/helpers/generateGroupSummaries.ts

```diff
@@ -0,0 +1,34 @@
+import extractTextFromDraftString from 'parabol-client/utils/draftjs/extractTextFromDraftString'
+import getRethink from '../../../database/rethinkDriver'
+import OpenAIServerManager from '../../../utils/OpenAIServerManager'
+import {DataLoaderWorker} from '../../graphql'
+
+const generateGroupSummaries = async (
+  meetingId: string,
+  dataLoader: DataLoaderWorker,
+  facilitatorUserId: string
+) => {
+  const [reflections, reflectionGroups, facilitator] = await Promise.all([
+    dataLoader.get('retroReflectionsByMeetingId').load(meetingId),
+    dataLoader.get('retroReflectionGroupsByMeetingId').load(meetingId),
+    dataLoader.get('users').loadNonNull(facilitatorUserId)
+  ])
+  if (!facilitator.featureFlags.includes('aiSummary')) return
+  const r = await getRethink()
+  const manager = new OpenAIServerManager()
+  for (const group of reflectionGroups) {
+    const reflectionsByGroupId = reflections.filter(
+      ({reflectionGroupId}) => reflectionGroupId === group.id
+    )
+    if (reflectionsByGroupId.length <= 1) continue
```

Comment:
If there's only one reflection in the group, I don't think there's any value in summarising it. Doing this will reduce our API costs too